src/_cffi_src/build_openssl.py

```diff
@@ -56,7 +54,7 @@ def _extra_compile_args(platform):
     function signatures in OpenSSL have changed from long -> unsigned long
     in the past. Since that isn't a precision issue we don't care.
     When we drop support for CRYPTOGRAPHY_OPENSSL_LESS_THAN_110 we can
-    revisit this.
+    revisit this. XXXX: Do we revisit now?
```

Comment:
Yes let’s file an issue to follow up on this and see if we can just pass this in CFLAGS/fix some definitions and remove this hack.